crown-court-proceeding/src/main/resources/schemas/apiCheckCrownCourtActionsResponse.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "apiCrownCourtActionsResponse.json",
+  "type": "object",
+  "title": "Crown court Request",
+  "description": "Data contract for crown court action response",
+  "properties": {
+    "repOrderDecision": {
+      "type": "string",
+      "title": "Rep Order decision"
+    },
+    "ccRepType": {
+      "type": "string",
+      "title": "crown court rep type"
+    },
+    "ccRepOrderDate": {
+      "type": "string",
+      "title": "crown court withdrawal date",
+      "format": "date-time"
```

Comment:
Can we simplify the names of the fields? It seems redundant to prepend each with cc when they're in the Crown Court Response object